torchvision/csrc/io/decoder/gpu/demuxer.h

```diff
@@ -15,11 +15,10 @@ class Demuxer {
   bool bMp4H264, bMp4HEVC, bMp4MPEG4;
   unsigned int frameCount = 0;
   int iVideoStream;
-  int64_t userTimeScale = 0;
   double timeBase = 0.0;
 
  public:
-  Demuxer(const char* filePath, int64_t timeScale = 1000 /*Hz*/) {
```

Comment:
Since this is part of the public constructor, are you confident that it's not going to affect anything on FBcode side?

Reply:
Yeah, this hasn't been exposed internally yet.